.github/workflows/screenshots.yml

```diff
@@ -137,42 +108,65 @@ jobs:
     runs-on: macos-latest
 
     steps:
-    - uses: actions/checkout@v1
-    - uses: actions/setup-ruby@v1
-      with:
-        ruby-version: '2.6.x'
-    
-    - name: Install Bundler
-      run: gem install bundler
-    
-    - name: Restore Ruby Dependency Cache
-      id: restore-ruby-dependency-cache
-      uses: actions/cache@v1
-      with:
-        path: vendor/bundle
-        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-    
-    - name: Install Ruby Dependencies
-      run: |
-        bundle config path vendor/bundle
-        bundle install --jobs 4 --retry 3
-
-    - name: Install Fastlane Dependencies
-      run: bundle exec fastlane run configure_apply
-
-    - name: Download Generated Screenshots
-      run: |
-        brew install awscli
-        cd fastlane
-        aws s3 cp s3://$S3_BUCKET/$GITHUB_RUN_ID/screenshots screenshots/  --recursive --exclude "*.html"
-    
-    - name: Generate and Upload Screenshot Summary
-      run: |
-        bundle exec fastlane create_screenshot_summary
-        aws s3 cp fastlane/screenshots/screenshots.html s3://$S3_BUCKET/$GITHUB_RUN_ID/screenshots/screenshots.html
-
-    - name: Archive Raw Screenshots
-      uses: actions/upload-artifact@v1
-      with:
-        name: raw-screenshots
-        path: fastlane/screenshots
+      - uses: actions/checkout@v1
+
+      - name: Install Native Dependencies
+        run: |
+          brew install imagemagick@6
+          brew link imagemagick@6 --force
```

Comment:
Yeah, good question – I was in a rush and this worked fine. IIRC we've done some work on the release toolkit to support v7, but pinning the dependency in this way prevents it from breaking in a future v8 version some day too.

I agree this might be good to come back to, but yeah probably in a future PR!

Reply:
Yeah I actually updated imagemagick to v7 during my trial when I worked on Android Screenshots back in the day. Which means that the release toolkit itself does depend on imagemagick 7, so it might not be great to depend on 2 different versions for different projects sharing the toolkit…

I think we ultimately need to unify and migrate this to v7 to be aligned with release-toolkit, at least. But that could be done later if that's too much work to include in that PR and if we want to merge this sooner than later.

Reply:
Ah ok awesome – in that case I've bumped it to v7 in 3a58fd0161419487ae9f969fca94d7f0fb6bd11f – let's see how it works!